cs/v1/CS_v1.0.0.yaml

```diff
@@ -419,7 +419,7 @@ paths:
           description: The carrier specific service code to filter by. The result will only return schedules including the service code.
         - schema:
             type: string
-            pattern: ^SR\d{5}[A-Z]$
+            pattern: '^SR\d{5}[A-Z]$'
             maxLength: 8
```

Comment:
A `minLength:8` should also be added as the pattern only allows for exactly 8 character input